presto-elasticsearch/pom.xml

```diff
@@ -303,20 +303,18 @@
         <dependency>
             <groupId>org.testcontainers</groupId>
             <artifactId>elasticsearch</artifactId>
-            <version>1.12.3</version>
             <scope>test</scope>
         </dependency>
 
         <dependency>
             <groupId>io.prestosql</groupId>
             <artifactId>presto-client</artifactId>
-            <scope>test</scope>
         </dependency>
 
+
         <dependency>
             <groupId>io.prestosql</groupId>
             <artifactId>presto-main</artifactId>
-            <scope>test</scope>
```

Comment:
I understand this is just temporary change.
This will need to be reverted, but let's focus on this once you get things working.

(https://github.com/prestosql/presto/pull/3347#discussion_r403839987)

Reply:
I have to do this to get IpAddressType

Reply:
i added a fixup for this -- https://github.com/prestosql/presto/pull/3347/commits/701d2f57b35e991e0280e78a8d33eed0e7921e96

Reply:
@findepi  there are problems with that change
`
IpAddressDecoder.java:[20,25] package io.prestosql.type does not exist
ElasticsearchPageSource.java:[73,32] package io.prestosql.type does not exist
`
the problematic imports are
`
io.prestosql.type.IpAddressOperators
io.prestosql.type.IpAddressType
`

the change you add to ElasticSearchMetadata should be replicated on this two class?

Reply:
fixed